app/src/main/java/org/simple/clinic/appconfig/CountryModule.kt

```diff
@@ -40,4 +42,26 @@ object CountryModule {
       appConfigRepository.saveDeployment(deployment)
     }
   }
+
+  private fun createV2CountryIfOldCountryIsPresent(appConfigRepository: AppConfigRepository) {
+    val selectedOldCountry = appConfigRepository.currentCountry().toNullable()
```

Comment:
Given that the old `Country` structure will be needed _exactly once_ at the time of migration and never again, I think we can plan for removing getting the old country via the repository and instead manually parse it using Moshi in this one migration path so that we don't end up using the method to read the old country anywhere else. What do you think?

Reply:
Yeah, we can do that, we can also just mark the function as deprecated to avoid usage. I personally prefer `deprecated` since I don't have to make many changes 😂 . But let me know what works for you.

Reply:
Deprecated indicates we plan to remove it. Removal is not what we are intending here since we need to older code to migrate old installations. So I would prefer to do the local parsing and conversion.

Reply:
If it's fine, I am gonna open a different PR that combines this and the removal of `Country_Old` data class. wdyt?

Reply:
Yeah, no issues with that.